src/views/v2/emails/template-parts/body/order/attendees-table/attendee-info.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Event Tickets Emails: Order Attendee Info
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/emails/template-parts/body/order/attendee-info.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link https://evnt.is/tickets-emails-tpl Help article for Tickets Emails template files.
+ *
+ * @version TBD
+ *
+ * @since TBD
+ *
+ * @var Tribe_Template  $this  Current template object.
+ * @var Module           $provider              [Global] The tickets provider instance.
+ * @var string           $provider_id           [Global] The tickets provider class name.
+ * @var array            $order                 [Global] The order object.
+ * @var int              $order_id              [Global] The order ID.
+ * @var bool             $is_tec_active         [Global] Whether `The Events Calendar` is active or not.
+ */
+
+if ( empty( $attendee ) ) {
+	return;
+}
+
+?>
+<tr class="tec-tickets__email-table-content-order-attendee-info-row">
+	<td class="tec-tickets__email-table-content-order-attendee-info tec-tickets__email-table-content-align-left" align="left">
+		<div><?php esc_html_e( $attendee['name'] ); ?></div>
+		<div><?php esc_html_e( $attendee['email'] ); ?></div>
```

Comment:
Should we make a safe check to see if they keys exist in case there's a particular case where these don't exist? (it'll break up the email functionality.

Reply:
Good call. I actually split each of them out into their own template file and bailed if they were empty.